### ISSUE TITLE
The mean cross-entropy loss function in attribute tagging does not converge

### ISSUE THREAD

**Issue by vitor-vfm** [ECE493Group3/personal-stylist-NN#4]
Even after 3000+ steps, the loss keeps alternating between 0.050 and 0.070. We need a new loss function
